decidim-comments/app/frontend/comments/comment_order_selector.component.jsx

```diff
@@ -0,0 +1,26 @@
+import { Component } from 'react';
+import { I18n }      from 'react-i18nify';
+
+/**
+ * A simple static component with the comment's order selector markup
+ * @todo Needs a proper implementation
```

Comment:
I guess this means that comments aren't actually ordered. Can you create an issue for this?